src/main/java/bit/javaoop/Employee.java

```diff
@@ -11,4 +11,14 @@ public Employee(String name, double salary) {
     public boolean isSatisfied() {
         return getSalary() > 10_000;
     }
-}
+
+
+    public String toStr(int line) {
+        String result = new String();
+        for(int i=0;i<line;i+=1)
```

Comment:
Now you have this whole logic of printing with a proper indentation duplicated in `Employee` and `AbstractManager` classes :) As I said, it would be better to extract it and move it to a separate class :)